Integrations/PagerDuty/PagerDuty.py

```diff
@@ -38,6 +38,8 @@
 '''SUFFIX ENDPOINTS'''
 GET_SCHEDULES_SUFFIX = 'schedules'
 CREATE_INCIDENT_SUFFIX = 'incidents'
+GET_INCIDENT_SUFFIX = 'incidents/'
+GET_SERVICES_SUFFIX = 'services?offset={}'
```

Comment:
better to pass `offset` in a dictionary as `params` so the requests module will handle correct escaping (not necessary here but for future reference)